src/palace/manager/util/datetime_helpers.py

```diff
@@ -11,15 +13,26 @@
 # https://docs.python.org/3/library/datetime.html#aware-and-naive-objects
 
 
-def datetime_utc(*args, **kwargs) -> datetime.datetime:
-    """Return a datetime object but with UTC information from pytz.
-    :return: datetime object
-    """
-    kwargs["tzinfo"] = pytz.UTC
-    return datetime.datetime(*args, **kwargs)
+P = ParamSpec("P")
+T = TypeVar("T")
+
+
+def _wrapper(func: Callable[P, T]) -> Callable[P, T]:
+    @wraps(func)
+    def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
+        kwargs["tzinfo"] = pytz.UTC
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+datetime_utc = _wrapper(datetime.datetime)
```

Comment:
Converted this to use a decorator, because it lets us type check the wrapped function, which is nice, since datetime.datetime takes a bunch of parameters.